source/test/TestApp/AutoGenTests/AutoGenDataTestData.cpp

```diff
@@ -3,12 +3,34 @@
 #include "TestApp.h"
 #include "AutoGenDataTests.h"
 #include "XAsyncHelper.h"
+#include "JsonUtils.h"
+
+#include <HttpClient.h>
+#include <httpClient/httpClient.h>
 
 namespace PlayFabUnit
 {
 
 using namespace PlayFab::Wrappers;
 
+HRESULT UploadFileSync(std::string url, PlayFab::JsonValue& requestBody)
```

Comment:
At the moment this InitiateFileUpload->FinalizeFileUpload flow is the only one included in XPlatC that relies on an outside HTTP operation, so this is just living here on it's own. Andy mentioned there may be other APIs that could need something like this, but they're not in yet, so this can be turned into something more generally suitable later if need be.